### PR TITLE
New package: mit-scheme-c-9.2

### PR DESCRIPTION
**Pull request by abenson**
Closes https://github.com/void-linux/void-packages/issues/900

### CODE REVIEW

File: srcpkgs/mit-scheme-c/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+version=9.2
+revision=1
+build_wrksrc="src"
+short_desc="Implementation of the Scheme programming language"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://www.gnu.org/software/mit-scheme/"
+distfiles="${GNU_SITE}/mit-scheme/stable.pkg/${version}/mit-scheme-c-${version}.tar.gz"
+checksum=4f6a16f9c7d4b4b7bb3aa53ef523cad39b54ae1eaa3ab3205930b6a87759b170
+nocross=yes
```
Comment (maxice8) [inline srcpkgs/mit-scheme-c/template]: put the error message or a brief explaination

Comment (cr6git) [reply srcpkgs/mit-scheme-c/template]:
````
$ ./xbps-src -a aarch64 pkg mit-scheme-c
=> mit-scheme-c-9.2_1: running do_build ...
run_cmd: ./configure --prefix=/builddir/mit-scheme-c-9.2/src/boot-root --enable-native-code=c --disable-host-scheme-test
checking whether make sets $(MAKE)... yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for gcc... aarch64-linux-gnu-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... configure: error: in `/builddir/mit-scheme-c-9.2/src':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details
=> ERROR: mit-scheme-c-9.2_1: do_build: 'etc/make-liarc.sh --prefix=/usr' exited with 1
=> ERROR:   in do_build() at srcpkgs/mit-scheme-c/template:15
````

Comment (abenson) [reply srcpkgs/mit-scheme-c/template]: I was having trouble getting it to build with the host compiler and then compile itself using that built compiler.

Comment (cr6git) [reply srcpkgs/mit-scheme-c/template]: @abenson take a decision.

**Comment by abenson** [comment]
Looking back at the build servers, it looks like it was never scheduled for build. [#10537](https://build.voidlinux.org/builders/x86_64_builder/builds/10537) was the commit right before, and [#10538](https://build.voidlinux.org/builders/x86_64_builder/builds/10538) was right after. I can't find it ever being built

**Pull request merged**